src/content/translations/ja/use/index.md

```diff
@@ -3,39 +3,40 @@ title: イーサリアムを使う
 lang: ja
 ---
 
-# イーサリアムを使う {#use-ethereum-applications}
+# イーサリアムを使う
```

Comment:
This file shouldn't exist 😑  - it's not a file from v1.1: https://crowdin.com/project/ethereumfoundation/ja#

![Image 2020-09-28 at 11 19 57 AM](https://user-images.githubusercontent.com/8097623/94470515-a2924700-017c-11eb-904e-f0bc8ef9f4f1.png)